apps/Gruntfile.js

```diff
@@ -427,7 +427,10 @@ testsContext.keys().forEach(testsContext);
     'layouts/_terms_interstitial':  './src/sites/studio/pages/layouts/_terms_interstitial.js',
     'maker/setup':                  './src/sites/studio/pages/maker/setup.js',
     'scriptOverview':               './src/sites/studio/pages/scriptOverview.js',
-    'home/teacher_homepage':        './src/sites/studio/pages/home/teacher_homepage.js'
+    'home/_teacher_homepage':       './src/sites/studio/pages/home/_teacher_homepage.js',
+    'home/_recent_courses':         './src/sites/studio/pages/home/_recent_courses.js',
+    'home/_link_blocks':            './src/sites/studio/pages/home/_link_blocks.js',
```

Comment:
Good point.  I was thinking these could be reusuable components, hence having their own entrypoints.  But as it happens, I plan to soon use React components built by @Erin007 , so will leave this intact until it's replaced.